elements/rh-dialog/rh-dialog.ts

```diff
@@ -0,0 +1,67 @@
+import { html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { classMap } from 'lit/directives/class-map.js';
+import { PfeModal } from '@patternfly/pfe-modal';
+import { RHDSScreenSizeController } from '../../lib/RHDSScreenSizeController.js';
+
+import './rh-context-provider.js';
+
+import styles from './rh-dialog.css';
+
+/**
+ * Dialog
+ */
+@customElement('rh-dialog')
+export class RhDialog extends PfeModal {
+  static readonly version = '{{version}}';
+
+  static readonly styles = [...PfeModal.styles, styles];
+
+  protected static closeOnOutsideClick = true;
+
+  #screenSize = new RHDSScreenSizeController(this);
+
+  @property({ reflect: true }) type?: 'video';
+
+  @property({ reflect: true, type: Boolean }) open = false;
+
+  render() {
+    const { mobile } = this.#screenSize;
+    return html`
+      <div id="rhds-wrapper" class=${classMap({ mobile })}>
+        ${super.render()}
+      </div>
+    `;
+  }
+
+  protected override renderHeaderSlot() {
+    return html`
+      <rh-context-provider color-palette="lightest">
+        <slot name="header"></slot>
+      </rh-context-provider>
```

Comment:
Just double checking that we still need this now that we've made the dialog trigger a sibling of the dialog component.  Could this work as an alternative?

```html
<rh-band color-palette="dark">
  <rh-dialog color-palette="lightest" trigger="trigger">
    <div slot="header">...</div>
    <div slot="content">...</div>
    <div slot="footer">...</div>
  </rh-dialog>
  <rh-button id="trigger">Open Dialog></div>
</rh-band>
```

Reply:
don't need it anymore. will loop back to put `color-palette` on dialog if and when it becomes a req